README.md

```diff
@@ -1,70 +1,51 @@
-# Getting Started with Create React App
+# Metrics-webapp
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+A web application that checks a list of metrics created the data fetched from a covid 19 API for countries in North America.This project applies HTML, CSS, and JavaScript best practices using React/Redux library and is fully responsive.
 
-## Available Scripts
+## live demo
```

Comment:
Great job doing your video demo, however, please also add the live demo of your application deployed to either Netlify or Heroku